portugues/colaborando.md

```diff
@@ -43,3 +43,5 @@ Vende seu PC?
 Um tijolo pesa 1 quilo mais meio tijolo. Quanto pesa 1 tijolo e meio?
 
 BR BR BR Hue hue hue!!!!
+
+Você é viciado em doces, não consegue pensar em uma resposta descente sem satisfazer sua quota diária de açúcar. Você procura a Chocolândia mais próxima de sua localização.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.